docs/faststore/docs/getting-started/1-onboarding/overview.mdx

```diff
@@ -8,10 +8,9 @@ This document provides an overview of the FastStore WebOps app, a tool designed
 
 <Steps>
 
-### Request the app's installation
-
-Please open a ticket with the [VTEX support team](https://help.vtex.com/support) to request the installation of the FastStore WebOps app.
+### Install the app
```

Comment:
We also have to update the naming on https://developers.vtex.com/docs/guides/faststore/1-onboarding-starting-the-project

Reply:
Fixed: https://github.com/vtexdocs/dev-portal-content/pull/1504/commits/69b40bb9bbc9566dff15137a5b244bfb2fd992aa